packages/core/pluggableElementTypes/models/baseTrackConfig.ts

```diff
@@ -38,6 +38,11 @@ export function createBaseTrackConfig(pluginManager: PluginManager) {
           'list of which feature attributes to index for text searching',
         defaultValue: ['Name', 'ID', 'Description'],
       },
+      textSearchIndexingFeatureTypesToExclude: {
```

Comment:
IMO this is a quite a verbose attribute name. do we need a "textIndexing" subconfig?

Reply:
this way more concise names could be used inside it

Reply:
I see that already we have textSearchAdapter subconfig for example, what if it was just moved inside a textSearchAdapter?

Reply:
A track textSearching/indexing subconfig could work if we really want to group all of that together, and would have to consider how to add [these](https://github.com/GMOD/jbrowse-components/pull/1881#issuecomment-895593278) to the subconfig once  the command implements track specific text search adapters. 
These are needed by the indexer, could you elaborate on how these could go in the text search adapters? Would we need to read aggregate text search adapters and track specific text search adapter configs before they are created?